reference/conanfile/attributes/binary_model.inc

```diff
@@ -74,9 +74,27 @@ If you want to do a safe check of settings values, you could use the ``get_safe(
         # Will be the default version if the return is None
         build_type = self.settings.get_safe("build_type", default="Release")
 
-The ``get_safe()`` method will return ``None`` if that setting or sub-setting doesn't
+The ``get_safe()`` method returns ``None`` if that setting or sub-setting doesn't
 exist and there is no default value assigned.
 
+It's also feasible to check the possible values defined for any setting/sub-setting using the ``possible_values()``
+method:
+
+.. code-block:: python
+
+    def generate(self):
+        definition = self.settings.compiler.version.possible_values()
+        self.output.info("HOST: " + ", ".join(definition))
+        definition = self.settings_build.compiler.version.possible_values()
+        self.output.info("BUILD: " + ", ".join(definition))
+        cppstds = self.settings.compiler.cppstd.possible_values()
+        self.output.info("CPPSTDS: " + str(cppstds))
+
+
+This method is layer-based so doing ``self.settings.possible_values()`` returns the whole :ref:`reference_config_files_settings_yml`
```

Comment:
I think we should better not use that 'layer-based' expression and maybe extending the example on how it works a bit

Reply:
Agree

Reply:
Now, it's explained according to the example above.